### PR TITLE
Create update-cart

### PR DESCRIPTION
**Pull request by MrEssCee**
Tutorial for Umbraco 13 Commerce to update cart

## Description
Tutorial to update Commerce cart
_What did you add/update/change?_

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [X] New content
* [ ] Updates related to a new version
* [ ] Other

## Deadline (if relevant)
ASAP
_When should the content be published?_
ASAP

### CODE REVIEW

**Comment by sofietoft** [comment]
Thanks for the PR @MrEssCee !

We'll get this reviewed as soon as possible.

File: 13/umbraco-commerce/how-to-guides/update-cart [hunk 1]
```diff
@@ -0,0 +1,203 @@
+---
+description: How-To update your cart.
+---
+
+Once an item has been added to the cart you would most likely need the functionality to update the cart.
+
+## Update shopping cart
+You will need a new page for the user to navigate to giving a summary of what's already in the cart and the ability to update each item.
+
+Create a new View, we will call it `cart.cshtml` with the below code
```
Comment (umbracotrd) [inline 13/umbraco-commerce/how-to-guides/update-cart:10]:
```suggestion
Create a new Document With Template, we will call it "Cart Page" with the template looks like below:
```

File: 13/umbraco-commerce/how-to-guides/update-cart [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+{
+    @foreach (var item in currentOrder.OrderLines.Select((ol, i) => new
+    {
+        OrderLine = ol,
+        Index = i
+    }))
+    {
+        <p>
+            @Html.Hidden($"orderLines[{item.Index}].Id", item.OrderLine.Id)
+            @item.OrderLine.Name @Html.Hidden($"orderLines[{item.Index}].Quantity", (int)item.OrderLine.Quantity, new { @type = "number" })
```
Comment (umbracotrd) [inline 13/umbraco-commerce/how-to-guides/update-cart:37]:
```suggestion
            @item.OrderLine.Name | @Html.TextBox($"orderLines[{item.Index}].Quantity", (int)item.OrderLine.Quantity, new { @type = "number" })
```

**Review by umbracotrd — COMMENTED** [review]
fix the syntax error in the code

**Review by sofietoft — COMMENTED** [review]
Thanks a bunch for the PR @MrEssCee ! 💪 

I've taken the liberty to fix some grammar and style guide issues.

Also, could you look into adding the code snippets to code blocks, using the following docs: https://docs.gitbook.com/content-editor/blocks/code-block#representation-in-markdown
Don't mind about the line numbers and the wrap, but adding the file name would be great 🙌

**Comment by sofietoft** [comment]
Also realized that the file name here is a bit wrong.
It needs to have the `.md` file extension to work with our docs 🙌

File: 13/umbraco-commerce/how-to-guides/update-cart.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
+---
+description: How to update your cart.
+---
+
+# Update Card
+
+Functionality is needed to update the cart once an item has been added. In this guide, you can learn how to add this functionality. 
+
+You need a new page for the user to give a summary of what is already in the cart and the ability to update each item.
```
Comment (sofietoft) [inline 13/umbraco-commerce/how-to-guides/update-cart.md:9]: Suggestion to shorten the sentence: 

```suggestion
You need a new page to summarize the items in the cart and allow users to update each item.
```

File: 13/umbraco-commerce/how-to-guides/update-cart.md [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+
+    public class OrderLineQuantityDto
+    {
+        public Guid Id { get; set; }
+        public decimal Quantity { get; set; }
+    }
+```
+
+{% hint style="warning" %}
+`ProductReference` is not used here but is an example of how you could pass the product reference to the controller for other similar related tasks.
```
Comment (sofietoft) [inline 13/umbraco-commerce/how-to-guides/update-cart.md:160]: Suggestion to shorten the sentence:

```suggestion
The `ProductReference` is not used here, but it is an example of how to pass the product reference to the controller for similar tasks.
```

**Review by sofietoft — APPROVED** [review]

**Pull request merged**